src/utils/FileRecordTools/FileReaders/SingleLineDelimTextFileReader.cpp

```diff
@@ -187,5 +187,5 @@ CHRPOS SingleLineDelimTextFileReader::getVcfSVlen() {
 		}
     }
     // not found
-    return INT_MIN;
+	return 1;
```

Comment:
As long as other place in bedtools doesn't handles negative length (which means returning `INT_MIN` would only causing interger overflow which cause unpredictable bugs rather than anything we expected), I believe return `INT_MIN` doesn't make sense anyway. Alternatively, we can turn this into an error, but I am not sure if this is actually valid VCF